setup.cfg

```diff
@@ -13,7 +13,8 @@ tests = True
 test=pytest
 
 [tool:pytest]
-addopts = --verbose --cov=yellowbrick --flakes --spec
+# TODO: add --spec and --verbose back to addopts
+addopts = --cov=yellowbrick --flakes
```

Comment:
This goes back to a pure pytest implementation of the test, but we lose the nice test output and verbosity. I want to add this back in when we can.

Reply:
Noted!